tests/test_metrics/test_subset_metric.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+import warnings
+
+import pandas as pd
+import numpy as np
+
+from sklearn.dummy import DummyClassifier
+from sklearn.metrics import accuracy_score
+from sklearn.pipeline import make_pipeline
+from sklearn.model_selection import GridSearchCV
+
+from sklego.preprocessing import ColumnSelector
+
+from sklego.metrics import subset_score
+
+
+class DisabledCV:
```

Comment:
Wouldn't setting the seed on an existing CV give the same result?

Reply:
That is true but not guaranteed to stay exactly the same if they change the way they use the RNG. Plus it's harder to reason about because you have to see the actual output before understanding what it will. But I'm ok with changing this, what do you think?